hydroflow_datalog_core/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "hydroflow_datalog_core"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+path = "src/lib.rs"
+
+[features]
+default = [ "diagnostics" ]
+diagnostics = [ "hydroflow_lang/diagnostics" ]
```

Comment:
If I understand this PR just lets us run `hydroflow_datalog_core` because the proc-macro stuff is in the old `hydroflow_datalog`?

Reply:
Yeah, so that we can call out to the insides of the proc macro for the WASM demo